www/source/partials/docs/_reference-environment-vars.html.md.erb

```diff
@@ -12,6 +12,7 @@ This is a list of all environment variables that can be used to modify the opera
 | `HAB_BLDR_URL` | build system, Supervisor | `https://bldr.habitat.sh` | Sets an alternate default endpoint for communicating with Builder. Used by the Habitat build system and the Supervisor |
 | `HAB_DOCKER_OPTS` | build system | no default | When running a Studio on a platform that uses Docker (macOS), additional command line options to pass to the `docker` command. |
 | `HAB_INTERNAL_BLDR_CHANNEL` | build system, Supervisor, exporters | `stable` | Channel from which Habitat-specific packages (e.g., `core/hab-sup`, `core/hab-launcher`, etc.) are downloaded on-demand when first called. Generally of use only for those developing Habitat. Only applies to Habitat-specific packages, and nothing else. |
+| `HAB_LISTEN_CTL` | build system, Supervisor, exporters | no default | Used to accept the [Chef EULA](https://docs.chef.io/chef_license.html#chef-eula). See [Accepting the Chef License](https://docs.chef.io/chef_license_accept.html#habitat) for valid values. |
```

Comment:
Did you mean `HAB_LICENSE` here?

Reply:
:doh: Yes, clearly copied from the line below